lib/classes/PluginManager.test.js

```diff
@@ -5,7 +5,7 @@ const PluginManager = require('../../lib/classes/PluginManager');
 const Serverless = require('../../lib/Serverless');
 const CLI = require('../../lib/classes/CLI');
 const Create = require('../../lib/plugins/create/create');
-
+const md5 = require('md5');
```

Comment:
Tried it before, but kept getting `hash update failed` error. Searched online and seemed to be a common issue. Didn't want to waste time investigating further when I can just use the super lightweight md5 module. It was made to keep things simple.